Makefile.in

```diff
@@ -130,10 +130,12 @@ else
   LIB_FUZZING_ENGINE = standalone_fuzz_target_runner.o
   ifeq (1,$(WITH_SANITIZERS))
   CFLAGS := $(CFLAGS) $(SANITIZERS_CFLAGS)
+  SFLAGS := $(SFLAGS) $(SANITIZERS_CFLAGS)
   LDFLAGS := $(LDFLAGS) $(SANITIZERS_CFLAGS)
   else
```

Comment:
Could we clean up the unbalanced indentation before/after else at the same time?